### PR TITLE
feat: policy docs

### PR DESCRIPTION
**Pull request by olenagerasimova**
part of #1237 

Prepared documentation for the policy.

### CODE REVIEW

**Comment by olenagerasimova** [comment]
@dgarus please, check. This is wiki correction, should be merged after we release artipie. Also, maybe everyone should review this PR to be aware of policy-related settings?

File: .wiki/Configuration-Policy.md [hunk 1]
```diff
@@ -0,0 +1,154 @@
...
+      - "*" # any action is allowed
+    maven-repo:
+      - install
+      - deploy
+    john/python-repo:
+      - read
+```
+`adapter_basic_permission` is the [permission type name](https://github.com/artipie/http/blob/master/src/main/java/com/artipie/security/perms/AdapterBasicPermissionFactory.java).
+This type is the permission type for any repository except for docker. Permission
+config of the `adapter_basic_permission` is the set of repository names with action list.
```
[the remark this answers is not in the document]
Comment (olenagerasimova) [reply .wiki/Configuration-Policy.md:111]: docker permissions are not implemented yet, I'll add description as soon as they be ready

**Review by andpopov — APPROVED** [review]
@olenagerasimova Thanks!

File: .wiki/Configuration-Credentials.md [hunk 2]
```diff
@@ -1,60 +1,70 @@
...
 
 ```yaml
 meta:
   credentials:
-    - type: file
-      path: _credentials.yml
+    - type: artipie
+      storage: 
+        type: fs
+        path: /tmp/artipie/sequrity
```
Comment (dgarus) [inline .wiki/Configuration-Credentials.md:13]: `sequrity` -> `security`

File: .wiki/Configuration-Credentials.md [hunk 3]
```diff
@@ -1,60 +1,70 @@
...
     - type: keycloak
       url: http://localhost:8080
       realm: realm_name
       client-id: client_application_id
       client-password: client_application_password
+  policy:
+    type: artipie
+    storage:
+      type: fs
+      path: /tmp/artipie/sequrity
```
Comment (dgarus) [inline .wiki/Configuration-Credentials.md:25]: `sequrity` -> `security`

File: .wiki/Configuration-Policy.md [hunk 4]
```diff
@@ -0,0 +1,154 @@
...
+      url: http://localhost:8080
+      realm: realm_name
+      client-id: client_application_id
+      client-password: client_application_password
+  policy:
+    type: artipie
+    eviction_millis: 180000 # optional, default 3 min
+    storage: # required
+      type: fs
+      path: /tmp/artipie/sequrity
```
Comment (dgarus) [inline .wiki/Configuration-Policy.md:27]: sequrity -> security

File: .wiki/Configuration-Policy.md [hunk 5]
```diff
@@ -0,0 +1,154 @@
...
+```
+`adapter_basic_permission` is the [permission type name](https://github.com/artipie/http/blob/master/src/main/java/com/artipie/security/perms/AdapterBasicPermissionFactory.java).
+This type is the permission type for any repository except for docker. Permission
+config of the `adapter_basic_permission` is the set of repository names with action list. 
+The following actions and synonyms are supported:
+- read, r, download, install, pull
+- write, w, publish, push, deploy, upload
+- delete, d, remove 
+
+Action `delete` in not supported by each adapter, check specific adapter docs for more details.
```
Comment (dgarus) [inline .wiki/Configuration-Policy.md:117]: To note can be used '>' symbol.
> Action `delete` in not supported by each adapter, check specific adapter docs for more details.

**Comment by olenagerasimova** [comment]
@dgarus thanks, all corrected

**Review by dgarus — APPROVED** [review]

**Review by andpopov — DISMISSED** [review]
@olenagerasimova Thanks!

**Review by ChGen — APPROVED** [review]

**Comment by olenagerasimova** [comment]
@dgarus @andpopov added docker permissions description, please, check

**Review by andpopov — DISMISSED** [review]
@olenagerasimova Thanks a lot!

**Comment by olenagerasimova** [comment]
@dgarus wiki is fully ready, please, check

**Review by dgarus — APPROVED** [review]
Thanks!

**Pull request merged**